### ISSUE TITLE
Error on startup: "The ordinal 116 could not be located in the dynamic link library mir_app.mir"

### ISSUE THREAD

**Issue by c8c8** [miranda-ng/miranda-ng#1088]
### Expected behavior
Startup with no error messages.

### Actual behavior
"The ordinal 116 could not be located in the dynamic link library mir_app.mir"

### Does it work in stable version?
I don't use stable.  I'm presently using 0.95.8 build #19074 (33ebf6b30f) x64. 

### Steps to reproduce that behavior
```
CPU: AMD Phenom(tm) II X6 1055T Processor [AMD64 Family 16 Model 10 Stepping 0] [DEP Enabled] [6 CPUs]
Installed RAM: 8192 MBytes
Microsoft Windows 7 Home Premium Edition, 64-bit Service Pack 1 (build 7601)
Internet Explorer: 9.11.9600.18860 (build 99600)
Administrator privileges: Yes
OS Languages: (UI | Locale (User/System)) : English/English | English/English
Free disk space on Miranda partition: 47597 MBytes

Miranda NG Version: 0.95.8 alpha build #19074 (33ebf6b30f) x64
Build time: 31 Dec 2017 4:08:10
Profile: C:\Program Files\Miranda NG\Profiles\Home\Home.dat (dbx mmap driver)
Profile size: 1310720 Bytes
Profile creation date: 1 Aug 2015 0:34:05
Language pack: No language pack installed
Service Mode: No

Active Plugins (19):
¤ AdvaImg.dll v.[b]0.11.0.2 [18 Dec 2017 12:18:30] - Miranda image services 
¤ AVS.dll v.0.95.3.4 [30 Dec 2017 4:08:12] - Avatar service 
¤ Clist_modern.dll v.0.9.1.7 [30 Dec 2017 4:10:24] - Modern contact list 
¤ CrashDumper.dll v.0.0.5.2 [12 Dec 2017 4:10:46] - Crash dumper 
¤ Dbx_mmap.dll v.0.95.3.1 [31 Dec 2017 4:08:16] - Miranda NG mmap database driver 
¤ Facebook.dll v.0.4.0.1 [30 Dec 2017 4:10:08] - Facebook RM 
¤ GG.dll v.0.11.0.3 [30 Dec 2017 4:08:36] - Gadu-Gadu protocol 
¤ ICQ.dll v.0.11.3.6 [30 Dec 2017 4:08:24] - IcqOscarJ protocol 
¤ Import.dll v.0.95.4.3 [11 Dec 2017 17:21:54] - Import contacts and messages 
¤ IRC.dll v.0.95.7.2 [30 Dec 2017 4:08:40] - IRC protocol 
¤ Jabber.dll v.0.11.0.5 [30 Dec 2017 4:09:08] - Jabber protocol 
¤ MSN.dll v.0.14.0.2 [30 Dec 2017 4:09:12] - MSN protocol 
¤ mTextControl.dll v.0.8.0.1 [11 Dec 2017 17:22:52] - Miranda text control 
¤ OpenSSL.dll v.0.1.0.2 [11 Dec 2017 17:26:56] - OpenSSL SSL API module 
¤ PluginUpdater.dll v.0.1.2.5 [24 Dec 2017 4:11:18] - Plugin updater 
¤ Popup.dll v.2.1.1.12 [27 Dec 2017 14:37:46] - Popup plus 
¤ SkypeWeb.dll v.0.12.2.5 [30 Dec 2017 4:13:40] - Skype protocol (Web) 
¤ TabSRMM.dll v.3.5.0.8 [27 Dec 2017 14:36:54] - TabSRMM 
¤ TopToolBar.dll v.0.8.0.4 [11 Dec 2017 17:23:56] - TopToolBar 

Unloadable Plugins (2):
¤ AIM.dll v.0.0.0.0 [18 Dec 2017 12:16:50] - <unknown> 
    Plugin statically linked to missing module: mir_core.mir
    Plugin statically linked to missing module: mir_app.mir

¤ DbChecker.dll v.0.0.0.0 [31 Dec 2017 4:11:04] - <unknown> 
    Plugin statically linked to missing module: mir_core.mir
    Plugin statically linked to missing module: mir_app.mir

Protocols and Accounts:
-------------------------------------------------------------------------------
Facebook                 0 - Enabled 1 - Disabled  Loaded
GG                       0 - Enabled 0 - Disabled  Loaded
ICQ                      0 - Enabled 0 - Disabled  Loaded
IRC                      0 - Enabled 0 - Disabled  Loaded
JABBER                   0 - Enabled 0 - Disabled  Loaded
MSN                      6 - Enabled 0 - Disabled  Loaded
SKYPE                    0 - Enabled 1 - Disabled  Loaded
MetaContacts             1 - Enabled 0 - Disabled  Not Loaded

Icon Packs:
-------------------------------------------------------------------------------
 Proto_AIM.dll [18 Dec 2017 12:23:06]
 Proto_Facebook.dll [30 Dec 2017 4:14:30]
 Proto_GG.dll [30 Dec 2017 4:14:32]
 Proto_ICQ.dll [30 Dec 2017 4:14:30]
 Proto_IRC.dll [30 Dec 2017 4:14:30]
 Proto_Jabber.dll [30 Dec 2017 4:14:30]
 Proto_MetaContacts.dll [31 Dec 2017 4:14:50]
 Proto_MSN.dll [30 Dec 2017 4:14:32]
 Proto_Skype.dll [30 Dec 2017 4:14:34]
 Proto_Weather.dll [24 Oct 2017 15:43:06]
 Proto_Yahoo.dll [23 Sep 2016 3:07:04]
 TabSRMM_icons.dll [27 Dec 2017 14:43:00]
 Toolbar_icons.dll [30 Dec 2017 4:14:30]
 xStatus_ICQ.dll [30 Dec 2017 4:14:32]
 xStatus_Jabber.dll [30 Dec 2017 4:14:32]
```

Everything worked fine until the nightly update before the one I'm using now.

**Comment by dartraiden** [comment]
Try to manually delete AIM.dll. PluginUpdater should remove it automatically, but we forgot to do that

**Comment by dartraiden** [comment]
Fixed in 671587fcdbbff6c13b5c09101d691c38d77170a8
